index.html

```diff
@@ -2641,61 +2644,64 @@ <h1>Servient Implementation</h1>
             and how they correspond to the abstract WoT Architecture (see <a href="#sec-wot-architecture"></a>).
             When implementing these concepts,
             a more detailed view is necessary that takes certain technical aspects into account.
-            This section presents the detailed architecture of a <a>Servient</a> implementation.
+            This section describes the detailed architecture of a <a>Servient</a> implementation.
         </p>
         <p>
-            <a href="#architecture-implementation"></a> shows a <a>Servient</a> implementation using the (optional) <a>WoT Scripting API</a> building block.
-            Here, the <a>WoT Runtime</a>, which manages the WoT-specific aspects,
-            is also a scripting runtime system that is able to interpret and execute application scripts.
-            <a>Servients</a> with the <a>WoT Scripting API</a> usually run on powerful devices, edge nodes, or in the cloud.
-            There are, however, also scripting environments for resource-constrained systems.
-            Furthermore, the WoT Architecture does not limit the application-facing API of the <a>WoT Runtime</a> to JavaScript/ECMAScript.
+            <a href="#architecture-implementation"></a> shows a <a>Servient</a> implementation that is using the (optional) <a>WoT Scripting API</a> building block.
+            Here, the <a>WoT Runtime</a> is also a <a>Scripting Runtime</a> system that,
+            in addition to managing the WoT-specific aspects, also interprets and executes the application scripts.
+            <a>Servients</a> that support the <a>WoT Scripting API</a> usually run on powerful devices, edge nodes, or in the cloud.
+            The WoT Architecture does not limit the application-facing API of the <a>WoT Runtime</a> to JavaScript/ECMAScript [[ECMAScript]].
+            There are also other runtime systems for resource-constrained systems that can be used to implement a <a>Servient</a>.
```

Comment:
This now sounds as if other runtime systems are only for resource-constrained systems... how about:

"Also other runtime systems can be used to implement a <a>Servient</a>."

We have examples and mentioning of resource constraints in the next paragraph, with is the detailed version of the last line.

Reply:
👍